src/core/QueryManager.ts

```diff
@@ -104,6 +104,19 @@ interface TransformCacheEntry {
 import type { DefaultOptions } from "./ApolloClient.js";
 import { Trie } from "@wry/trie";
 import { AutoCleanedWeakCache, cacheSizes } from "../utilities/index.js";
+export interface QueryManagerOptions<TStore> {
```

Comment:
```suggestion

export interface QueryManagerOptions<TStore> {
```

Don't hate me for this one 🙈. Could we add a space between the `import` and `export` statements?